### PR TITLE
Convert to a plugin

### PR DESCRIPTION
**Pull request by gitting**
The two first commits should be squashed prior to pulling.
Well, all of them, actually.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -15,16 +14,8 @@ internals.init = function () {
...
-        method: 'GET',
-        path: '/version',
-        config: {
-            description: 'Returns the version of the server',
-            handler: function (request, reply) {
-
-                return reply({ version: Package.version });
-            }
-        }
+    server.register({ register: require('./version') }, function (err) {
```
Comment (chyld) [inline lib/index.js]: move module loading to top of file

File: lib/version.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                return reply({ version: Package.version });
+            }
+        }
+    });
+
+    next();
+};
+
+exports.register.attributes = {
+    pkg: Package
```
Comment (hueniverse) [inline lib/version.js]: Internal plugins should just have `name`.

**Comment by hueniverse** [comment]
See general comments on issue.

**Comment by gitting** [comment]
Is there a naming convention for internal plugins? Should `multiple: true` ever be used for internal plugins `attributes`?

**Comment by hueniverse** [comment]
I usually use the file name as the plugin name unless there is conflict. As for `multiple`, that's based on the nature of the plugin.

**Pull request closed without merging**